AirCasting/SessionViews/ShareSession/ShareSessionViewModel.swift

```diff
@@ -5,36 +5,45 @@ import Foundation
 
 enum ShareSessionError: Error {
     case noSessionURL
+    case requestError
 }
 
 protocol ShareSessionViewModel: ObservableObject {
     var streamOptions: [ShareSessionStreamOptionViewModel] { get set }
     var alert: AlertInfo? { get set }
     var showShareSheet: Bool { get set }
+    var showInvalidEmailError: Bool { get set }
     var sharingLink: URL? { get set }
+    var email: String { get set }
     func didSelect(option: ShareSessionStreamOptionViewModel)
-    func shareLinkTepped()
+    func shareLinkTapped()
+    func shareEmailTapped()
     func cancelTapped()
     func sharingFinished()
+    func isEmailValid() -> Bool
```

Comment:
I think this is only used internally, so it should not be exposed in a `protocol`